js_modules/dagit/packages/core/src/partitions/PartitionStatus.tsx

```diff
@@ -336,6 +337,33 @@ export const PartitionStatus: React.FC<{
   );
 };
 
+export const PartitionStatusCountsOnly: React.FC<{
+  small?: boolean;
+  partitionNames: string[];
+  partitionStateForKey: (key: string) => PartitionState;
+}> = ({small, partitionNames, partitionStateForKey}) => {
+  const byState = countBy(partitionNames, partitionStateForKey);
+
+  const counts = Object.entries(byState)
+    .filter(([, count]) => count > 0)
+    .map(([state, count]) => `${count.toLocaleString()} ${state}`)
+    .join(', ');
```

Comment:
I hope this will fit. Any common circumstances where it might not?

Reply:
Hmm I will double check before I merge this, that's a good question actually. I think there can be three states that we need to display.